src/3d/qgscameracontroller.cpp

```diff
@@ -795,3 +796,58 @@ void QgsCameraController::moveView( float tx, float ty )
   mCameraPose.setCenterPoint( center );
   updateCameraFromPose( true );
 }
+
+bool QgsCameraController::willHandleKeyEvent( QKeyEvent *event )
+{
+  if ( event->key() == Qt::Key_QuoteLeft )
+    return true;
+
+  switch ( mCameraNavigationMode )
+  {
+    case WalkNavigation:
+    {
+      switch ( event->key() )
+      {
+        case Qt::Key_Left:
+        case Qt::Key_A:
+        case Qt::Key_Right:
+        case Qt::Key_D:
+        case Qt::Key_Up:
+        case Qt::Key_W:
+        case Qt::Key_Down:
+        case Qt::Key_S:
+        case Qt::Key_PageUp:
+        case Qt::Key_E:
+        case Qt::Key_PageDown:
+        case Qt::Key_Q:
```

Comment:
@nyalldawson , we need the tilt (~) as well as ctrl+~ in there too. Ctrl+~ is assigned to open the python console over here (can't remember if it's a default shortcut or not)

Reply:
That's already handled at the top of the function by catching Qt::Key_QuoteLeft